src/app/SBE/table/Table.js

```diff
@@ -36,9 +45,9 @@ const DataTable = ({ parentCode, mapSearch, passedComponents = [], userCode }) =
 
   if (!tableData)
     return (
-      <Text pt={5} px={10}>
-        {'No data found.'}
-      </Text>
+      <Center>
```

Comment:
What is the reason for replacing the text to indicate no data found with a loading spinner?

Reply:
as seen in [LOJ-675](https://gada.atlassian.net/browse/LOJ-675), the 'no data found' message was being displayed while data was being loaded

Reply:
The fix is to not render the no data found in middle of rendering a table, which is a bug. The ticket doesn’t mean you change the ui when there is no data, the way you have implemented this means even when there is no data in the table it will always show a loading spinner which is another bug in itself.